stf_functional_tests.yml

```diff
@@ -65,9 +65,10 @@
       ansible.builtin.import_role:
         name: test_verify_email
 
-    - name: "Test observabilityStrategy"
-      ansible.builtin.import_role:
-        name: test_observability_strategy
+# Disable observability strategy run
```

Comment:
If you're going to comment this out, add a short explanation as to why it's commented out.

Otherwise, remove the lines completely

I'm okay with either option